site/source/docs/porting/Debugging.rst

```diff
@@ -170,7 +170,7 @@ returns the ``what`` function call result.
     emscripten::function("getExceptionMessage", &getExceptionMessage);
   };
 
-This requires using the linker flag ``--bind``.
+This requires using the linker flags ``-l embind -sEXPORT_EXCEPTION_HANDLING_HELPERS``.
```

Comment:
Sorry, can you also remove the space after the `-l`?   Normally libraries are linked like `-lc` or `-lsql`

Reply:
:+1: done